### DISCUSSION TITLE
Armour Punch

### DISCUSSION THREAD

**Discussion by LouderthanThunder** [Lambosaurus/hives-and-torches#26] [Feats]
You are proficient at using armour piecing weapons. For every crit you achieve roll an additional d4 and add it to your total damage 

Prerequisites -

**Comment by Lambosaurus** [comment]
This bucks the trend of critical effects consuming the crit. But if it did it would be too weak.

Could it be advantage against armoured foes with piercing weapons or something like that?

**Reply by LouderthanThunder** [reply]
I wanted to make it cascading. You can keep triggering crits. Like a knight with his sword pushing it into another knights visor or something

**Reply by Lambosaurus** [reply]
Uh, this could be a runaway process. Once you have a 1,2,3, and 4, then every d4 is a crit.

**Reply by Lambosaurus** [reply]
I like the idea, but the core math is pretty broken. This will need some workshopping.
